src/taipy/gui/server.py

```diff
@@ -110,6 +111,8 @@ def my_index(path):
                     styles=styles,
                     version=version,
                 )
+            if path == "status.json":
```

Comment:
Do you think it would be safer to have a 'taipy.status' path instead?
That both better protects user-defined page names and allows for some extensions in the future...

Reply:
ok for taipy.status.json :-)